src/FwdState.h

```diff
@@ -184,6 +184,7 @@ class FwdState: public RefCountable, public PeerSelectionInitiator
         bool dont_retry;
         bool forward_completed;
         bool destinationsFound; ///< at least one candidate path found
+        bool tunneled; ///< \copydoc Security::EncryptorAnswer::tunneled
```

Comment:
```C++
/// whether we spliced the connections instead of negotiating encryption
bool tunneled;
```

That EncryptorAnswer field description does not work well in FwdState context -- FwdState does not (or should not) know much about TLS negotiations and splicing.

How about the following?

```suggestion

        /// Security::PeerConnector dispatched this request (instead of returning control to us)
        bool tunneled;
```